blockprod/src/interface/blockprod_interface.rs

```diff
@@ -33,7 +33,6 @@ pub trait BlockProductionInterface: Send {
     async fn generate_block(
         &mut self,
         input_data: Option<GenerateBlockInputData>,
```

Comment:
This shouldn't be optional anymore, right?

Reply:
I think it would have to be `None` for `RequiredConsensus::IgnoreConsensus` though

Reply:
Well, then why not add a third arm in `GenerateBlockInputData` with index 0 to cover that? It'll have the same effect of `None`

Reply:
Of course. Cool, doing that now